_posts/2019-05-20-foreman-community-newsletter---may-2019.md

```diff
@@ -0,0 +1,175 @@
+and audit. Sounds like good news to us! All the work that we have been putting
+in is being noticed :)
+
+Not to forget, we had the very awesome yellow Foreman hats distributed at the Foreman
+booth. Thanks to Tomer and Ori for handling the design and shipment of the goodies.
+
+### Community Survey analysis is now out
+
+Thanks to everyone who filled out this year's community survey - we have now
+the analysis of the report that was prepared by Greg. We had 116 replies, which is a
+little down on last year, but still more than enough data to work with. As promised, Greg has
+written us a [blog](https://theforeman.org/2019/05/2019-foreman-survey-analysis.html) on the
+data recieved from the community survey :)
+
+To get an complete overview of the survey you can watch the survey analysis explained by
+Greg at the [Foreman Community Demo](https://youtu.be/2a2sjVFyqTQ?t=1482).
+
+Few of the key points from the survey were:
+
+1) New contributors to the Foreman community have almost doubled since the last year, thats a great news!
+2) Forum has been a the most common mode of discussion in the community.
+3) A drop is seen in the use of Debian as a Operating System used for Foreman installation
+whereas we see a hike in the Centos user base.
+4) We asked a question to the community wherein we displayed four different login screens and asked
+for which one looked the best and as you can see the Foreman login page is change to the one which
+was the favored by the community :)
+
+### [Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the "Add To Calendar" button
+on specific events if you don't wish to import the whole thing.
+
+* [Forman Birthday Event](https://community.theforeman.org/t/foreman-birthday-party/13731)
+
+Since 10 years Foreman gives system administrators the power to easily automate repetitive tasks, quickly deploy applications, and proactively manage servers. It’s time to provision your travel luggage and configure your calendars to alert you on July 25, 2019! Save the date for the 10th Foreman Birthday Party organized by NETWAYS, ATIX and the Foreman Project! Find more information/updates on the event in this
+[blog](https://www.netways.de/blog/2019/05/22/10th-foreman-birthday-event/?utm_content=buffer4787f&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer).
+
+* [OpenExpo Europe](https://community.theforeman.org/t/openexpo-europe/13467)
+
+Tomer will be giving a talk about “Hybrid cloud management with Foreman” at the conference.
+He will be talking about some of the main features of Foreman, and how you can leverage it to simplify and unify your datacenter management. All the best Tomer :)
+
+* [Puppet Contributor Summit](https://community.theforeman.org/t/puppet-contributor-summit/13985)
```

Comment:
This was last week, no point in promoting it now (and afaik there weren't any foreman devs there)